src/renderer/components/AircraftSection/LiveryConversion/index.tsx

```diff
@@ -151,10 +151,10 @@ const ConversionProgress: React.FC = () => {
     });
 
     return (
-        <div className="flex flex-row gap-x-5 items-center">
+        <div className="flex flex-row gap-x-2 items-center">
             <Progress className="w-full" showInfo={false} percent={(completedConversions / (conversionsToPerform + completedConversions)) * 100} trailColor="#ef4444" strokeColor="white" />
 
-            <span className="text-xl text-white">{completedConversions / (conversionsToPerform + completedConversions) * 100}%</span>
+            <span className=" w-16 text-xl text-white text-right">{(completedConversions || conversionsToPerform) ? completedConversions / (conversionsToPerform + completedConversions) * 100 + '%' : '0%'}</span>
```

Comment:
I would put the content of that `span` on a newline

Reply:
well yeah okay, agree it got quite long